Formula/m/mariadb.rb

```diff
@@ -45,9 +46,12 @@ class Mariadb < Formula
   uses_from_macos "ncurses"
   uses_from_macos "zlib"
 
+  on_macos do
+    depends_on "openjdk" => :build
+  end
+
   on_linux do
     depends_on "linux-pam"
-    depends_on "readline" # uses libedit on macOS
```

Comment:
Has been unused since `readline` v5 has incompatible license. Only links if using `-DNOT_FOR_DISTRIBUTION=ON`, which is not something we should do. So, should use `libedit` instead (already a dep).